documentation/asciidoc/microcontrollers/c_sdk/official_sdk.adoc

```diff
@@ -1,4 +1,4 @@
-== Raspberry Pi Pico C/{cpp} SDK
+== Raspberry Pi Pico C/C++ SDK
```

Comment:
Bit of historical context - in some situations we **needed** to use `{cpp}` instead of `C++` because of the way asciidoc uses `++` to mark https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/ , so if `C++` was used twice within one paragraph then things went badly wrong :rofl: 

But it's possible that past-lurch was a bit overzealous in swapping `C++` to `{cpp}` in places that it wasn't actually needed? :wink: